README.md

```diff
@@ -41,14 +41,14 @@ I modified `configure` script to ignore the terminal library check.  It's OK sin
 GUI frontend for Wasm is always used instead of CUI frontend. I needed many
 workarounds to pass `configure` checks.
 
-emscripten provides Unix-like environment. So `os_unix.c` can support Wasm. However,
-some features are not supported by emscripten. I added many `#ifdef FEAT_GUI_WASM`
+emscripten provides Unix-like environment. So `os_unix.c` can support Wasm. Some 
+features are not supported by emscripten. I added many `#ifdef FEAT_GUI_WASM`
 guards to disable features which cannot be supported by Wasm (i.e. `fork (2)`
 support, PTY support, signal handlers are stubbed, ...etc).
 
 I created `gui_wasm.c` heavily referencing `gui_mac.c` and `gui_w32.c`. Event loop
 (`gui_mch_update()` and `gui_mch_wait_for_chars()`) is simply implemented with
-`sleep()`. And almost all UI rendering events arer passed to JavaScript layer
+`sleep()`. All UI rendering events are passed to JavaScript layer
```

Comment:
Could you update this line as follows? I think original one no longer represents current architecture correctly.

```markdown
`sleep()`.  Rendering UI to `<canvas>` is done in JavaScript layer
```

Could you also consider that max number of characters per one line is 80?